### ISSUE TITLE
paste-after: TypeError: 'NoneType' object is not reversible

### ISSUE THREAD

**Issue by cool-RR** [saulpw/visidata#1889]
If you start VisiData 2.11 in a folder and press `p`, you get this error:

```
 Traceback (most recent call last):
   File "/home/ramrachum/.local/lib/python3.10/site-packages/visidata/basesheet.py", line 187, in execCommand
     escaped = super().execCommand2(cmd, vdglobals=vdglobals)
   File "/home/ramrachum/.local/lib/python3.10/site-packages/visidata/basesheet.py", line 74, in execCommand2
     exec(code, vdglobals, LazyChainMap(vd, self))
   File "paste-after", line 1, in <module>
   File "/home/ramrachum/.local/lib/python3.10/site-packages/visidata/clipboard.py", line 117, in paste_after
     to_paste = list(deepcopy(r) for r in reversed(vd.memory.cliprows))
 TypeError: 'NoneType' object is not reversible
```

I assume that `paste-after` can't be used because nothing was copied, but if so, the exception message should reflect that.

**Comment by anjakefala** [comment]
This is actually already fixed on `develop`! @midichef fixed it with fb441708ce4b0a319f42beb3fe3f37375cfa3d0c

**Comment by cool-RR** [comment]
Oops, should have checked.
